### PR TITLE
added test2.java

### PR DESCRIPTION
**Pull request by ToastyGull**
this adds the message "wow" to the files so cool and epic amazing!!! XD

### CODE REVIEW

File: test2.java [hunk 1]
```diff
@@ -0,0 +1 @@
+#wow
```
Comment (Timothy-A-Chan) [inline test2.java:1]: Why is this here?

**Comment by ToastyGull** [comment]
hghg

**Review by Timothy-A-Chan — COMMENTED** [review]
hm

File: test2.java (hunk 1 above)
Comment (ToastyGull) [reply test2.java:1]: to take up space
